datamodel/changelogs/0001/14_default_values.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE tww_od.default_values
```

Comment:
You mention a table in tww_sys in your PR description and it is stored in tww_od in your code.
-> tww_od : any user can edit these values
-> tww_vl : a manager user can edit these values
-> tww_sys : an administrator user can edit these values
- [ ] Depending on who will be able to define these default values, the good schema has to be chosen.

Reply:
I moved it from sys to od because @urskaufmann has use cases where a tww_user needs the right to alter the values